### ISSUE TITLE
README says `npm install` but library not published to npm

### ISSUE THREAD

**Issue by benkaiser** [parshap/music-organizer#1]
It isn't published, at least under the name in the README.md

**Comment by parshap** [comment]
Yeah this is not published to npm despite what the readme says. I haven't gotten it to a place where it's really usable by others, but hopefully one day I will.

**Comment by benkaiser** [comment]
No worries :)
